bin/utils/buildFonts.js

```diff
@@ -52,6 +54,43 @@ async function buildFonts(targetFile) {
 
   const fonts = [...SYSTEM_FONTS, ...googleFonts.items.map(normalizeFont)];
 
+  const tempFile = 'temp.ttf';
+  for (let i = 0, n = fonts.length; i < n; i++) {
+    const font = fonts[i];
+    if (font.fontFileURL) {
+      try {
+        // eslint-disable-next-line no-await-in-loop
+        const response = await got(font.fontFileURL);
```

Comment:
I have tried our `fetch.js` but couldn't make it work.

Reply:
No objections to using `got` if it solves issues. That also means I can retire `fetch.js`, so 👍